src/ralph/admin/helpers.py

```diff
@@ -0,0 +1,3 @@
+from django.apps import apps
+
+APP_MODELS = {model._meta.model_name: model for model in apps.get_models()}
```

Comment:
it's only one line so we can redefine it in sitetree. besides that - it's definition (APP_MODELS = {model._meta.model_name: model for model in apps.get_models()}) breaks when there will be model with the same name in more than one app